src/shared/slider/slider.css

```diff
@@ -36,7 +37,7 @@
     color: #fff;
     display: block;
     text-align: center;
-    z-index: 2;
+    z-index: 1;
     position: relative;
 }
 .react-multiple-carousel__arrow--left {
```

Comment:
Change to:

.react-multiple-carousel__arrow--left {
    top: 35%;
    left: 15px;
}
.react-multiple-carousel__arrow--left::before {
    content: '\1438';
}
.react-multiple-carousel__arrow--right {
    top: 35%;
    right: 15px;
}
.react-multiple-carousel__arrow--right::before {
    content: '\1433';
}